eth_utils/applicators.py

```diff
@@ -82,10 +82,17 @@ def apply_formatters_to_dict(
         if key in formatters:
             try:
                 yield key, formatters[key](item)
-            except (TypeError, ValueError) as exc:
-                raise type(exc)(
-                    "Could not format value %r as field %r" % (item, key)
-                ) from exc
+            except ValueError as exc:
+                new_error_message = "Could not format invalid value %r as field %r" % (
```

Comment:
Why is this using % formatting?  I expected `.format(...)` or f-string.

Reply:
eth-utils still seems to support python 3.5, so no f-strings, and I didn't think we had a defined preference for `.format(...)`, so my default was to leave it in the previous style.